src/docdb/editors/DocDBDocumentNodeEditor.ts

```diff
@@ -18,7 +18,8 @@ export class DocDBDocumentNodeEditor implements ICosmosEditor<RetrievedDocument>
         const collectionNode = this._documentNode.parent;
         const databaseNode = collectionNode.parent;
         const accountNode = databaseNode.parent;
-        return `${accountNode.treeItem.label}/${databaseNode.treeItem.label}/${collectionNode.treeItem.label}/${this._documentNode.treeItem.label}`;
+        const subscriptionNode = accountNode.parent;
+        return `${subscriptionNode.treeItem.label}/${accountNode.treeItem.label}/${databaseNode.treeItem.label}/${collectionNode.treeItem.label}/${this._documentNode.treeItem.label}`;
```

Comment:
nit: Just my opinion, but I don't think the subscription label provides any value here (since account names are already globally unique) and it just makes the message longer/harder to read